packages/client/tests/functional/_utils/relationMode/computeMatrix.ts

```diff
@@ -60,13 +60,32 @@ export function computeMatrix({ relationMode, providersDenyList }: ComputeMatrix
     },
   }
 
-  const providersMatrix = providerFlavors.map((providerFlavor) => ({
+  let providersMatrix = providerFlavors.map((providerFlavor) => ({
     provider: getProviderFromFlavor(providerFlavor),
     providerFlavor,
     id: 'String @id',
     relationMode,
   }))
 
+  const skipSQLServer = process.env.TEST_SKIP_MSSQL
+  const skipMongodb = process.env.TEST_SKIP_MONGODB
+  const skipCockroachdb = process.env.TEST_SKIP_COCKROACHDB
+  const skipVitess = process.env.TEST_SKIP_VITESS
+
+  // in "no-docker" tests we want to skip some databases
+  if (skipSQLServer) {
+    providersMatrix = providersMatrix.filter((it) => it.provider !== Providers.SQLSERVER)
+  }
+  if (skipMongodb) {
+    providersMatrix = providersMatrix.filter((it) => it.provider !== Providers.MONGODB)
+  }
+  if (skipCockroachdb) {
+    providersMatrix = providersMatrix.filter((it) => it.provider !== Providers.COCKROACHDB)
+  }
```

Comment:
Not a huge fan of this chain but hey, if it works 😅

Reply:
I didn't add it here though but where it was needed locally in the tests failing